samples/client/petstore/rust/hyper/petstore/src/models/duplicate_one_of.rs

```diff
@@ -15,6 +15,7 @@ use serde::{Deserialize, Serialize};
 #[serde(untagged)]
 pub enum DuplicateOneOf {
     Order(Box<models::Order>),
+    Order(Box<models::Order>),
```

Comment:
not sure why there are changes to rust samples

can you please file a new PR based on the latest master to see if these changes to rust samples no longer appear?

Reply:
please ignore

the issue is with the latest master and i'll try to fix it

Reply:
fixed